viam-cartographer/lua_files/map_builder.lua

```diff
@@ -0,0 +1,23 @@
+-- Copyright 2016 The Cartographer Authors
```

Comment:
these lua files are copy & pasted from https://github.com/cartographer-project/cartographer/tree/master/configuration_files so that unit tests can configure cartographer without needing to globally install the cartographer lua files.